src/scss/_objects.modules.scss

```diff
@@ -90,6 +90,18 @@
   }
 }
 
+.accordion-cta-image {
+  width: 270px;
+  height: 180px;
+  object-fit: cover;
+}
+
+.resource-image {
+  width: 100px;
+  height: 100px;
+  object-fit: cover;
```

Comment:
Using this just as a placeholder. `object-fit` doesn't work in IE nor Edge 😐 

I'll remove these when I build out the `Resources` components